installers/rancher/main.go

```diff
@@ -30,5 +30,5 @@ func kernelParams(j job.Job, s *ipxe.Script) {
 		s.Args("rancher.network.interfaces.eth2.dhcp=true")
 	}
 
-	s.Args("tinkerbell=${tinkerbell}")
+	s.Args("boots=${boots}")
```

Comment:
we don't manage the rancher script so need to keep kernel args the same as before